build.sh

```diff
@@ -74,6 +74,12 @@ BUILD_STATIC_FAISS=OFF
 INSTALL_PREFIX=${INSTALL_PREFIX:=${PREFIX:=${CONDA_PREFIX}}}
 PARALLEL_LEVEL=${PARALLEL_LEVEL:=""}
 
+# Allow setting arbitrary cmake args via the $CUML_ADDL_CMAKE_ARGS variable. Any
+# values listed here will override existing arguments. For example:
+# CUML_ADDL_CMAKE_ARGS="-DBUILD_CUML_C_LIBRARY=OFF" ./build.sh
```

Comment:
really picky thought but could we make it CUML_EXTRA_CMAKE_ARGS instead? like how in Make you have "EXTRA_CFLAGS" insteasd of ADDL_CFLAGS?

Reply:
I renamed it to `CUML_EXTRA_CMAKE_ARGS` as requested. There was also a similar variable for python build that I renamed to `CUML_EXTRA_PYTHON_ARGS` to match. See the help output for an example:
```sh
 The following environment variables are also accepted to allow further customization:
   PARALLEL_LEVEL         - Number of parallel threads to use in compilation.
   CUML_EXTRA_CMAKE_ARGS  - Extra arguments to pass directly to cmake. Values listed in environment
                            variable will override existing arguments. Example:
                            CUML_EXTRA_CMAKE_ARGS="-DBUILD_CUML_C_LIBRARY=OFF" ./build.sh
   CUML_EXTRA_PYTHON_ARGS - Extra argument to pass directly to python setup.py
```